### ISSUE TITLE
Support multiple WebJars packed into a single artifact.

### ISSUE THREAD

**Issue by SomeoneToIgnore** [vaadin/flow-maven-plugin#18]
Consider the artifact

```xml
<dependency>
    <groupId>com.vaadin</groupId>
    <artifactId>vaadin-charts-webjar</artifactId>
    <version>6.0.0-alpha3</version>
</dependency>
```

it contains multiple `bower.json` files which causes `flow-maven-plugin` to fail currently.
This should not happen: instead, two bower packages should be extracted

**Comment by johannesh2** [comment]
Sorry for pressing the wrong button. I was about to comment, that we can also make a release that contain only charts dependency (so only one bower.json)

**Comment by SomeoneToIgnore** [comment]
Have already created a fix: https://github.com/vaadin/flow-maven-plugin/pull/20 for the current structure, so it's up to you.

**Comment by johannesh2** [comment]
Thank you!

**Comment by johannesh2** [comment]
Actually good thing that you fixed it, since highcharts is available only as a classic webjar which doesn't play well with Flow and vaadin-charts-flow, so we can't go that route anyways.

**Comment by SomeoneToIgnore** [comment]
Fixed, version `0.0.4` is released.
